windows.js

```diff
@@ -46,6 +48,25 @@ export async function install(platform, ruby) {
   return rubyPrefix
 }
 
+// all standard msvc OpenSSL builds use C:\Program Files\Common Files\SSL
+// as per openssl/openssl
+function mswinSSL(hostedRuby) {
```

Comment:
Could we just set `SSL_CERT_FILE` instead like it's done for 2.2 and 2.3 above?

Reply:
With 2.2. and 2.3, the reason for using the ENV settings is that the OpenSSL package used pointed to a folder specific to the system the builder used.

The path for normal Windows OpenSSL builds is actually in the openssl/openssl code.  MSYS2 changes it to reference the location of the exe using OpenSSL.  Actions should have the files located there, but I don't want to assume that.

So, I'd prefer to set up OpenSSL as expected, and consider the 2.2 and 2.3 builds the exceptions.

Reply:
I would think creating an empty `C:\\Program Files\\Common Files\\SSL\\certs` directory has no effect, so I would just skip and only check the cert.pem.
I'd prefer to simplify and just check `C:\\Program Files\\Common Files\\SSL\\cert.pem` exists and otherwise throw.

If the cert.pem file was missing, would the `ruby -ropen-uri -e 'puts open(%{https://rubygems.org/}) { |f| f.read(1024) }'` check file? If so, I think we can use that as our check of "OpenSSL works and can check certificates", without any extra code in windows.js.

Reply:
> If the cert.pem file was missing, would the ruby -ropen-uri -e 'puts open(%{https://rubygems.org/}) { |f| f.read(1024) }'

It fails.  Just tried locally after renaming my cert file in 'C:\Program Files\Common Files\SSL`.

> C:\\Program Files\\Common Files\\SSL\\certs

It is a correctly set up OpenSSL, and people may not check for its existence...

Reply:
I think the check in the workflow is enough then.
If people rely on a directory being in the image, that's their responsibility.
`setup-ruby` should care about providing a working Ruby, including a working OpenSSL C extension, but not try to "lint" the whole system. There might be exceptions if we know something that is particularly unreliable, but for now I think the open-uri check above is good enough for testing OpenSSL.

Reply:
My concern was more about having all the Ruby builds setup the same.  If some correctly connect a cert file, all should.  Same for the directory.  I'll check.

Reply:
X509::DEFAULT_CERT_DIR exists on all builds, see:
https://github.com/MSP-Greg/ruby-setup-ruby-info/runs/460598415

head builds are failing, due to an issue with ruby/openssl.  Fixed there, not pushed to ruby/ruby yet.

Reply:
I see, it just fails if we don't setup those files: https://github.com/ruby/setup-ruby/runs/465758532